smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -5402,6 +5408,7 @@ StackInterpreter >> extSendSuperBytecode [
 	extA := 0.
 	BytecodeSetHasDirectedSuperSend ifTrue: [
 		extB >= 64 ifTrue: [
+			self druidForceIntepretation.
 			numArgs := (byte bitAnd: 7) + (extB - 64 << 3).
 			extB := 0.
 			numExtB := 0.
```

Comment:
What's the expected compiler code for the interpreter code after a `self druidForceIntepretation`? I understood that the execution continues in the interpreted (so the code below should not be meta-compiled?)

Reply:
Yes, the code below should be unreachable and considered as dead code. Yes.
So, as you say it should not, but that's an optimization so I did not implement it :)